src/icons.ts

```diff
@@ -0,0 +1,379 @@
+export default {
```

Comment:
Other idea to reduce filesize: just export the icon names as a type:

```ts
export type iconKeys = 'custom-inherited-disabled' |
    'custom-inherited' | 
    'custom-uninherited' | 
    'default-action-bookmark' | 
   ...
```

Then you can use `icon: iconKeys` as the type in the `src/ui/settings/index.ts` file.